backend/dal/Repositories/LeaseTermRepository.cs

```diff
@@ -29,6 +29,7 @@ public void Delete(long leaseTermId)
 
         public PimsLeaseTerm Update(PimsLeaseTerm pimsLeaseTerm)
         {
+            pimsLeaseTerm.PimsLeasePayments = null; //Do not edit lease payments.
```

Comment:
This is a conceptual problem with EF, but by default it wants to manipulate child relationships. Since we are trying to handle that explicitly, we need to come up with a strategy here.

Reply:
Instead of setting it to null  you can mark the property not to be modified lilke:
```
pimsLeaseTerm.Property(nameof(<property you want>)).IsModified = false;
```